

<!DOCTYPE html>
<html class="writer-html5" lang="ls -ltr" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Research &mdash; Tuttle and Associates - Beyond Warming 1.1.1 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>
        <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home" alt="Documentation Home"> Tuttle and Associates - Beyond Warming
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <!-- Local TOC -->
              <div class="local-toc"><ul>
<li><a class="reference internal" href="#">Research</a><ul>
<li><a class="reference internal" href="#key-areas-and-select-publications">Key areas and select publications</a></li>
<li><a class="reference internal" href="#application">Application</a></li>
<li><a class="reference internal" href="#evolution">Evolution</a></li>
<li><a class="reference internal" href="#systems">Systems</a></li>
<li><a class="reference internal" href="#interaction">Interaction</a></li>
<li><a class="reference internal" href="#function">Function</a></li>
<li><a class="reference internal" href="#structure">Structure</a></li>
<li><a class="reference internal" href="#infrastructure">Infrastructure</a></li>
</ul>
</li>
</ul>
</div>
            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Tuttle and Associates - Beyond Warming</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Research</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/Research.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="research">
<h1>Research<a class="headerlink" href="#research" title="Permalink to this headline">¶</a></h1>
<h2>CompBio research</h2><p>The general area of our scientific research is computational biology, genomics, and proteomics.
The goal is to elucidate processes responsible for protein, small molecule, nucleic acid, and interactome structure,
function, interaction, evolution, and design so as to understand (and reproduce by computing simulation) how the
information encoded by the genome of an organism specifies behavior and characteristics in the context of its environment.</p>
<p>Our research leads us to tackle computational problems in algorithmic studies of astronomically large spaces,
bioinformatics/data mining, and massively parallel and distributed computing. The work described in the publications is
generally encapsulated into a variety of webservers/applications/services (links included) and downloadable software.
A full reverse chronologically ordered list of the publications is available as part of my CV. More significant publications
are denoted by * along with other annotations such as an accompanying cover or introductory article signifying the
notability of a publication so that people may use this as a guide to help focus their studies.</p>
<div class="section" id="key-areas-and-select-publications">
<h2>Key areas and select publications<a class="headerlink" href="#key-areas-and-select-publications" title="Permalink to this headline">¶</a></h2>
<p>Structural and functional studies of biologically important proteins, systems, and problems. Use the structure and function prediction tools developed by us to help guide experimentalists in manipulating proteins and extracting information about their function and structure in vivo, both at the single molecule as well as at the genomic/systems levels. Some key areas include work on therapeutic (inhibitor) discovery and nanobiotechnology. This work is usually done in collaboration with experimentalists. I list these papers first since they demonstrate a true application of the work we do. In many cases, these are prospective verification (i.e., a prediction is made before the answer is known and verified).</p>
</div>
<div class="section" id="application">
<h2>Application<a class="headerlink" href="#application" title="Permalink to this headline">¶</a></h2>
<p>Structural and functional studies of biologically important proteins, systems, and problems. Use the structure and function prediction tools developed by us to help guide experimentalists in manipulating proteins and extracting information about their function and structure in vivo, both at the single molecule as well as at the genomic/systems levels. Some key areas include work on therapeutic (inhibitor) discovery and nanobiotechnology. This work is usually done in collaboration with experimentalists. I list these papers first since they demonstrate a true application of the work we do. In many cases, these are prospective verification (i.e., a prediction is made before the answer is known and verified).</p>

<h3> <a name="therapeutics">Therapeutics</a> </h3>

<ol>

  <li> Schuler J, Falls Z, Mangione W, Hudson M, Bruggemann L,
  <b>Samudrala R</b>. Evaluating performance of drug repurposing
  technologies. <cite>Drug Discovery Today</cite>
  2020. <i>invited</i>. *

  <li> Mangione W, Falls Z, Chopra G, <b>Samudrala R</b>. cando.py:
  Open source software for predictive bioanalytics of large scale
  drug-protein-disease data. <cite>Journal of Chemical Information and
  Modeling</cite> 2020. <i>in press</i>. *

  <li> Mangione W, Falls Z, Melendy T, Chopra G, <b>Samudrala R</b>.
  Shotgun drug repurposing biotechnology to tackle epidemics and
  pandemics. <cite>Drug Discovery Today</cite> 25: 1126-1128, 2020. *

  <li> Fine J, Konc J, <b>Samudrala R</b>, Chopra G. CANDOCK: Chemical
  Atomic Network-Based Hierarchical Flexible Docking Algorithm Using
  Generalized Statistical Potentials. <cite>Journal of Chemical
  Information and Modeling</cite> 60: 1509-1527, 2020. *

  <li> Fine J, Lackner R, <b>Samudrala R</b>, Chopra G. Computational
  chemoproteomics to understand the role of selected psychoactives in
  treating mental health indications. <cite>Scientific Reports</cite>
  9, 1315, 2019. *
  
  <li> Schuler J, <b>Samudrala R</b>. Fingerprinting CANDO: Increased
  accuracy with structure and ligand based shotgun drug
  repurposing. <cite>ACS Omega</cite> 4: 17393-17403, 2019.  *

  <li> Schuler J, Mangione W, <b>Samudrala R</b>, Ceusters
  W. Foundations for a realism-based drug repurposing
  ontology. Proceedings of the 10th International Conference on
  Biomedical Ontology, 2019.

  <li> Falls Z, Mangione W, Schuler J, <b>Samudrala R.</b> Exploration
  of interaction scoring criteria in the CANDO platform. <i>BMC
  Research Notes</i> 12: 318, 2019. *

  <li> Mangione W, <b>Samudrala R.</b> Identifying protein features
  responsible for improved drug repurposing accuracies using the CANDO
  platform: Implications for drug design. <i>Molecules</i> 24: 167,
  2019. *

  <li> Schuler J, Hudson M, Schwartz D, <b>Samudrala R</b>. A
  systematic review of computational drug discovery, development, and
  repurposing for Ebola Virus Disease Treatment. <i>Molecules</i> 22:
  E1777, 2017.

  <li> Chopra C, Kaushik S, Elkin PL, <b>Samudrala R</b>. Combating
  Ebola with repurposed therapeutics using the CANDO
  platform. <cite>Molecules</cite> 21: 1537, 2016. *

  <li> Craig JK, Risler JK, Loesch KA, Dong W, Baker D, Barrett LK,
  Subramanian S, <b>Samudrala R</b>, Van Voorhis WC. Mycobacterium
  cytidylate kinase appears to be an undruggable target. <cite>Journal
  of Biomolecular Design</cite> 21: 695-700, 2016.

  <li> Chopra G, <b>Samudrala R</b>. Exploring polypharmacology in
  drug discovery and repurposing using the CANDO
  platform. <cite>Current Pharmaceutical Design</cite> 22: 3109-3123
  2016.

  <li> Manocheewa S, Mittler JE, <b>Samudrala R</b>, Mullins
  JI. Composite sequence-structure stability models as screening tools
  for identifying vulnerable targets for HIV drug and vaccine
  development. <cite>Viruses</cite> 7: 5718-5735, 2015.
  
  <li> Sethi G, Chopra G, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2015a.pdf">Multiscale
  modelling of relationships between protein classes and drug behavior
  across all diseases using the CANDO platform</a>. <cite>Mini Reviews
  in Medicinal Chemistry</cite>, 15: 705-717, 2015.
  
  <li> Minie M, Chopra G, Sethi G, Horst J, White G, Roy A, Hatti K,
  <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2014b.pdf">CANDO and
  the infinite drug discovery frontier</a>.  <cite>Drug Discovery
  Today</cite> 19: 1353-1363, 2014. *

  <li> Lertkiatmongkol P, Assawamakin A, White G, Chopra G,
  Rongnoparut P, <b>Samudrala R</b>, Tongsima S. Distal effect of
  amino acid substitutions in CYP2C9 polymorphic variants causes
  differences in interatomic interactions against
  (S)-warfarin. <cite>PLoS One</cite> 8: e74053, 2013.

  <li> Strategic protein target analysis for developing drugs to stop
  dental caries.  Horst JA, Pieper U, Sali A, Zhan L, Chopra G,
  <b>Samudrala R</b>, Featherstone JD.  <cite>Advances in Dental
  Research</cite> 24: 86-93, 2012. *
 
  <li> Horst JA, Laurenzi A, Bernard B, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2012a.pdf">Computational
  multitarget drug discovery</a>. <cite>Polypharmacology</cite>
  263-301, 2012. *
  
  <li> Nicholson CO, Costin JM, Rowe DK, Lin L, Jenwitheesuk E,
  <b>Samudrala R</b>, Isern S, Michael SF. Viral entry inhibitors
  block dengue antibody-dependent enhancement <i>in
  vitro</i>. <cite>Antiviral Research</cite> 89: 71-74 2010. *
    
  <li> Movahedzadeh F, Balaubramanian V, Bernard B,
  Iyer S, <b>Samudrala R</b>, Franzblau SG, Balganesh TS.
  Anti-tuberculosis agents: A rational approach for discovery and
  development.  <cite>Genomic and computational tools for emerging
  infectious diseases</cite>, 2010.

  <li> Costin JM, Jenwitheesuk E, Lok S-M, Hunsperger E, Conrads KA,
  Fontaine KA, Rees CR, Rossmann MG, Isern S, <b>Samudrala R</b>,
  Michael SF. Structural optimization and <i>de novo</i> design of
  dengue virus entry inhibitory peptides. <cite>PLoS Neglected Tropical
  Diseases</cite> 4: e721, 2010. *
      
  <li> Bernard B, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2009g.pdf">A
  generalized knowledge-based discriminatory function for biomolecular
  interactions</a>.  <cite>Proteins: Structure, Function, and
  Bioinformatics</cite> 76: 115-128, 2009.
    
  <li> Jenwitheesuk E, Horst JA, Rivas K, Van Voorhis WC, <b>Samudrala
  R</b>. <a href="http://compbio.org/publications/samudrala_2008a.pdf">Novel paradigms
  for drug discovery: Computational multitarget
  screening</a>. <cite>Trends in Pharmacological Sciences</cite> 29:
  62-71, 2008. [Accompanying cover.] *

  <li> <b>Samudrala R</b>, Jenwitheesuk E. <a
  href="http://compbio.org/publications/samudrala_2007f.pdf">Identification of potential HIV-1
  targets of minocycline</a>. <cite>Bioinformatics</cite> 23:
  2797-2799, 2007.
  
  <li> Wang K, Mittler J, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2006c.pdf">Comment on "Evidence for
  positive epistatis in HIV-1"</a>. <cite>Science</cite> 312: 848b,
  2006.

  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2005l.pdf">Identification of potential
  multitarget antimalarial drugs</a>. <cite>Journal of the American
  Medical Association</cite> 294: 1490-1491, 2005. *
  
  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2005n.pdf">Heptad-repeat-2 mutations
  enhance the stability of the enfuvirtide-resistant HIV-1 gp41
  hairpin structure</a>. <cite>Antiviral Therapy</cite> 10: 893-900,
  2005. *

  <li> Jenwitheesuk E, Wang K, Mittler J, <b>Samudrala R</b>.
  <a href="http://compbio.org/publications/samudrala_2005e.pdf">PIRSpred: A webserver for
  reliable HIV-1 protein-inhibitor resistance/susceptibility
  prediction</a>. <cite>Trends in Microbiology</cite> 13: 150-151,
  2005.

  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2005c.pdf">Virtual screening of HIV-1
  protease inhibitors against human cytomegalovirus protease using
  docking and molecular dynamics</a>. <cite>AIDS</cite> 19: 529-533,
  2005.

  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2005b.pdf">Prediction of HIV-1
  protease inhibitor resistance using a protein-inhibitor flexible
  docking approach</a>. <cite>Antiviral Therapy</cite> 10: 157-166,
  2005.

  <li> Jenwitheesuk E, Wang K, Mittler J, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2004e.pdf">Improved accuracy of HIV-1
  genotypic susceptibility interpretation using a consensus
  approach</a>. <cite>AIDS</cite> 18: 1858-1859, 2004.

  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2003e.pdf">Identifying inhibitors of
  the SARS coronavirus proteinase</a>. <cite>Bioorganic & Medicinal
  Chemistry Letters</cite> 13: 3989-3992, 2003. [Most Cited Paper
  2003 - 2006 Award.] *
  
  <li> Jenwitheesuk E, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2003b.pdf">Improved prediction of
  HIV-1 protease-inhibitor binding energies by molecular dynamics
  simulations</a>. <cite>BMC Structural Biology</cite> 3: 2, 2003. *

  <li> Wang K, Jenwitheesuk E, <b>Samudrala R</b>, Mittler J.  <a
  href="http://compbio.org/publications/samudrala_2004c.pdf">Simple linear model
  provides highly accurate genotypic predictions of HIV-1 drug
  resistance</a>. <cite>Antiviral Therapy</cite> 9: 343-352, 2004.

  <li> Wang K, <b>Samudrala R</b>, Mittler J. <a
  href="http://compbio.org/publications/samudrala_2004b.pdf">Weak
  agreement between predictions of ``reduced susceptibility'' from
  Antivirogram and PhenoSense assays</a>. <cite>Journal of Clinical
  Microbiology</cite> 42: 2353-2354, 2004.

  <li> Wang K, <b>Samudrala R</b>, Mittler J. <a
  href="http://compbio.org/publications/samudrala_2004g.pdf">HIV-1
  genotypic drug resistance interpretation algorithms need to include
  hypersusceptibility mutations</a>. <cite>Journal of Infectious
  Diseases</cite> 190: 2055-2056, 2004.

  <li> Wang K, <b>Samudrala R</b>, Mittler J.  <a
  href="http://compbio.org/publications/samudrala_2004f.pdf">Antivirogram
  or PhenoSense: a comparison of their reproducibility and an analysis
  of their correlation</a>. <cite>Antiviral Therapy</cite> 9: 703-712,
  2004.

  <li> <a href="http://protinfo.org/pirspred/">Protein inhibitor
  resistance/susceptibility prediction (PIRSpred) web server
  module</a>
  
  <li> <a href="http://protinfo.org/cando/">Computational
  analysis of novel drug opportunities (CANDO) platform</a>
  
</ol>


<h3> Nanobiotechnology </h3>

<ol>

  <li> Cementomimetics-constructing a cementum-like biomineralized
  microlayer via amelogenin-derived peptides.  Gungormus M, Oren EE,
  Horst JA, Fong H, Hnilova M, Somerman MJ, Snead ML, <b>Samudrala
  R</b>, Tamerler C, Sarikaya M.  <cite>International Journal of Oral
  Sciences</cite> 2: 69-77, 2012. *
   
  <li> Notman R, Oren EE, Tamerler C, Sarikaya M, <b>Samudrala R</b>,
  Walsh TR. Solution study of engineered quartz binding
  peptides using replica exchange molecular dynamics.
  <cite>Biomacromolecules</cite> 11: 3266-3274, 2010.
    
  <li> Oren EE, Notman R, Kim IW, Evans J, Walsh T, <b>Samudrala
  R</b>, Tamerer C, Sarikaya M. Probing the molecular mechanisms of
  quartz-binding peptides. <cite>Langmuir</cite> 26: 11003-11009,
  2010.
    
  <li> <b>Samudrala R</b>, Oren EE, Cheng C, Horst, J, Bernard B,
  Gungormus M, Hnilova M, Fong H, Tamerler C, Sarikaya M. <a
  href="publications/samudrala_2008e.pdf">Knowledge-based design of
  inorganic binding peptides</a>. <cite>Proceedings of the conference
  on the Foundations of Nanoscience: Self-Assembled Architectures and
  Devices</cite>, 2008.
  
  <li> Evans JS, <b>Samudrala R</b>, Walsh TR, Oren EE, Tamerler
  C. <a href="publications/samudrala_2008d.pdf">Molecular design of
  inorganic-binding polypeptides</a>. <cite>MRS Bulletin</cite> 33:
  514-518, 2008. [<a
  href="publications/samudrala_2008d.introduction.pdf">Accompanying
  cover and introductory article with biographies on pages 504-512</a>.] *
   
  <li> Oren EE, Tamerler C, Sahin D, Hnilova M, Seker UOS, Sarikaya M,
  <b>Samudrala R</b>. <a href="publications/samudrala_2007g.pdf">A novel
  knowledge-based approach for designing inorganic binding
  peptides</a>. <cite>Bioinformatics</cite> 23: 2816-2822, 2007. *
      
</ol>

<h3> General and specific functional studies </h3>

<ol>
  <li> Bruggemann L, Hawthorne C, <b>Samudrala R</b>, Lopez-Campos
  GH. Linking genome and exposome: Computational analysis of human
  variation in chemical-target interactions. <cite>Student Health
  Technology Informatics</cite> 270: 1331-1332, 2020.

  <li> Mandloi S, Falls Z, Deng R, <b>Samudrala R</b>, Elkin
  PL. Association of C>U RNA editing with human disease
  variants. <cite>Student Health Technology Informatics</cite> 270:
  1205-1206, 2020.
  
  <li> Homo-dimerization and ligand binding by the leucine-rich repeat
  domain at RHG1/RFS2 underlying resistance to two soybean pathogens.
  Afzal AJ, Srour A, Goil A, Vasudaven S, Liu T, <b>Samudrala R</b>,
  Dogra N, Kohli P, Malakar A, Lightfoot DA.  <cite>BMC Plant
  Biology</cite> 13: 43, 2013.
   
  <li> Self-assembly of filamentous amelogenin requires calcium and
  phosphate: from dimers via nanoribbons to fibrils.  Martinez-Avila
  O, Wu S, Kim SJ, Cheng Y, Khan F, <b>Samudrala R</b>, Sali A, Horst
  JA, Habelitz S.  <cite>Biomacromolecules</cite> 13: 3494-502, 2012.

  <li> An P, Li R, Wang JM, Yoshimura T, Takahashi M, <b>Samudrala
  R</b>, O'Brien SJ, Phair J, Goedert JJ, Kirk GD, Troyer JL, Sezgin
  E, Buchbinder SP, Donfield S, Nelson GW, Winkler CA.  Role of exonic
  variation in chemokine receptor genes on AIDS: CCRL2 F167Y
  association with pneumocystis pneumonia.  <cite>PLoS Genetics</cite>
  7: e1002328, 2011.
  
  <li> Horst OV, Horst JA, <b>Samudrala R</b>, Dale BA.  Caries
  induced cytokine network in the odontoblast layer of human teeth.
  <cite>BMC Immunology</cite> 12: 9, 2011.

  <li> Cunningham ML, Horst JA, Rieder MJ, Hing AV, Stanaway IB, Park
  SS, <b>Samudrala R</b>, Speltz ML.  IGF1R variants associated with
  isolated single suture craniosynostosis. <cite>The American Journal
  of Human Genetics</cite> 155A: 91-97, 2011. [Accompanying cover.]

  <li> Borlee BR, Goldman AD, Murakami K, <b>Samudrala R</b>, Wozniak
  DJ, Parsek MR. <a
  href="publications/samudrala_2010b.pdf">Pseudomonas aeruginosa uses
  a cyclic-di-GMP-regulated adhesin to reinforce the biofilm
  extracellular matrix</a>. <cite>Molecular Microbiology</cite> 75:
  827-842, 2010.  [Accompanying cover.]
    
  <li> Goldman AD, Leigh JA, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009k.pdf">Comprehensive computational
  analysis of Hmd enzymes and paralogs in methanogenic
  Archaea</a>. <cite>BMC Evolutionary Biology</cite> 9: 199, 2009.

  <li> Jenkins C, <b>Samudrala R</b>, Geary S, Djordjevic SP.
  <a href="publications/samudrala_2008b.pdf">Structural and functional
  characterisation of an organic hydroperoxide resistance (Ohr)
  protein from <i>Mycoplasma gallisepticum</i></a>.
  <cite>Journal of Bacteriology</cite> 190: 2206-2208, 2008.
  
  <li> Chevance FFV, Takahashi N, Karlinsey JE, Gnerer J, Hirano T,
  <b>Samudrala R</b>, Aizawa S-I, Hughes KT.  <a
  href="publications/samudrala_2007e.pdf">The mechanism of outer
  membrane penetration by the eubacterial flagellum and implications
  for spirochete evolution</a>. <cite>Genes and Development</cite> 21:
  2326-2335, 2007.
  
  <li> Bockhorst J, Lu F, Janes JH, Keebler J, Gamain B, Awadalla P, Su
  X, <b>Samudrala R</b>, Jojic N, Smith JD.  <a
  href="publications/samudrala_2007c.pdf">Structural polymorphism
  and diversifying selection on the pregnancy malaria vaccine
  candidate VAR2CSA</a>. <cite>Molecular and Biochemical
  Parasitology</cite> 155: 103-112, 2007.
   
  <li> Berube PM, <b>Samudrala R</b>, Stahl DA. <a
  href="publications/samudrala_2007b.pdf">Transcription of
  <i>amoC</i> is associated with the recovery of <i>Nitrosomonas
  europaea</i> from ammonia starvation</a>. <cite>Journal of
  Bacteriology</cite> 89: 3935-3944, 2007.
   
  <li> Korotkova N, Le Trong I, <b>Samudrala R</b>, Korotkov K, Van
  Loy CP, Bui A-L, Moseley SL, Stenkamp RE. <a
  href="publications/samudrala_2006f.pdf">Crystal structure and mutational
  analysis of the DaaE adhesin of <i>Escherichia
  coli</i></a>. <cite>Journal of Biological Chemistry</cite> 281:
  22367-22377, 2006.
   
  <li> Howell DPG, <b>Samudrala R</b>, Smith JD. <a
  href="publications/samudrala_2006d.pdf">Disguising itself -
  insights into <i>Plasmodium falciparum</i> binding and immune
  evasion from the DBL crystal structure</a>. <cite>Molecular and
  Biochemical Parasitology</cite> 148: 1-9, 2006.
    
  <li> Wang W, Zheng H, Yang S, Yu H, Li J, Jiang H, Su J, Yang L,
  Zhang J, McDermott J, <b>Samudrala R</b>, Wang J, Yang H, Yu J,
  Kristiansen K, Wong GK, Wang J. <a
  href="publications/samudrala_2005k.pdf">Origin and evolution of new exons in
  rodents</a>. <cite>Genome Research</cite> 15: 1258-1264, 2005.
   
  <li> Liu T, Jenwitheesuk E, Teller D, <b>Samudrala R</b>.
  <a href="publications/samudrala_2005j.pdf">Structural insights into the Cellular
  Retinaldehyde Binding Protein (CRALBP)</a>. <cite>Proteins:
  Structure, Function, and Bioinformatics</cite> 61: 412-422, 2005.
   
  <li> Ekwa-Ekok C, Diaza GA, Carlson C, Hasegawad T,
  <b>Samudrala R</b>, Limf K, Yabug JM, Levya B, Schnapp LM.  <a
  href="publications/samudrala_2004i.pdf">Genomic organization and sequence
  variation of the human integrin subunit 8 gene
  (ITGA8)</a>. <cite>Matrix Biology</cite> 23: 487-496, 2004.

  <li> Wang J, Zhang J, Zheng H, Li J, Liu D, Li H, <b>Samudrala
  R</b>, Yu J, Wong GK.  <a href="publications/samudrala_2004h.pdf">Mouse
  transcriptome: Neutral evolution of "non-coding" complementary
  DNAs</a>. <cite>Nature</cite> 431, 2004.
  
  <li> Jenkins C, <b>Samudrala R</b>, Anderson I, Hedlund BP, Petroni
  G, Michailova N, Pinel N, Overbeek R, Rosati G, Staley JT.  <a
  href="publications/samudrala_2002f.pdf">Genes for the cytoskeletal protein
  tubulin in the bacteria genus
  <i>Prosthecobacter</i></a>. <cite>Proceedings of the
  National Academy of Sciences</cite> 99: 17049-17054, 2002.
    
  <li> Van Loy CP, Sokurenko EP, <b>Samudrala R</b>, Moseley S.
  <a href="publications/samudrala_2002c.pdf">Identification of a DAF binding
  domain in the Dr adhesin</a>. <cite>Molecular Microbiology</cite>
  45: 439-452, 2002.
  
  <li> <b>Samudrala R</b>, Xia Y, Levitt M, Cotton NJ, Huang ES, Davis R.
  <a href="publications/samudrala_2000a.pdf">Probing structure-function
  relationships of the DNA polymerase alpha-associated zinc-finger
  protein using computational approaches</a>. In Altman R, Dunker K,
  Hunter L, Klein T, Lauderdale K, eds. <cite>Proceedings of the
  Pacific Symposium on Biocomputing</cite> 179-189, 2000.

  <li> <a href="http://protinfo.org">Protinfo structure, function, and interaction prediction server</a>
</ol>

</div>
<div class="section" id="evolution">
<h2>Evolution<a class="headerlink" href="#evolution" title="Permalink to this headline">¶</a></h2>
<p>We use our prediction protocols to explore early evolution and origin of life issues.</p>
<ol>

  <li> Goldman AD, Barrows J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2012d.pdf">The enzymatic and metabolic
  capabilities of early life</a>. <cite>PLoS One</cite> 7: e39912,
  2012. *

  <li> Goldman AD, Horst JA, Hung L-H, <b>Samudrala R</b>. Evolution
  of the protein repertoire. <cite>Systems Biology</cite>: 207-237,
  2012.  (R Meyers, Editor. Wiley-VCH Wienheim, Germany.)
   
  <li> Goldman AD, <b>Samudrala R</b>, Barrows J.  <a
  href="publications/samudrala_2010a.pdf">The evolution and
  functional repertoire of translation proteins following the origin of
  life</a>. <cite>Biology Direct</cite> 5: 15, 2010. *
   
  <li> Goldman AD, Leigh JA, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009k.pdf">Comprehensive computational
  analysis of Hmd enzymes and paralogs in methanogenic
  Archaea</a>. <cite>BMC Evolutionary Biology</cite> 9: 199, 2009.
</ol>   
</div>
<div class="section" id="systems">
<h2>Systems<a class="headerlink" href="#systems" title="Permalink to this headline">¶</a></h2>
<p>Application and integration of single molecule structure and function prediction techniques
to whole genomes and proteomes in an integrated manner. Combine single molecule and
genomic/proteomic data to to explore the relationships among the molecular and organismal
(systems) worlds and create a comprehensive picture of the relationship between genotype
and phenotype.</p>
<ol>
  <li> Hung L-H, <b>Samudrala R</b>. Rice protein models from the
  Nutritious Rice for the World Project.  <i>bioRxiv 091975; doi: <a
  href="https://doi.org/10.1101/091975">https://doi.org/10.1101/091975</a></i>,
  2016.
  
  <li> Minie M. <b>Samudrala R</b>. <a
  href="publications/samudrala_2013d.pdf">The promise and challenge of
  digital biology</a>. <cite>Journal of Bioengineering and Biomedical
  Sciences</cite> 3: e118, 2013. <i>editorial</i>.
    
  <li> Matasci N, Hung L-H, ..., <b>Samudrala R</b>, Tian Z, Wu X, Sun
  X, Zhang Y, Wang J, Leebens-Mack J, Wong GSK. Data access for the
  1,000 Plants (1KP) project. <cite>Gigascience</cite> 3: 17, 2014.
  
  <li> McDermott J, Ireton R, Montgomery K, Bumgarner R, <b>Samudrala
  R</b> (editors).  <a href="publications/samudrala_2009f.pdf">Computational
  systems biology</a>.  <cite>Methods in Molecular Biology</cite> 541:
  <i>v</i>-<i>ix</i>, 2009. *
  
  <li> Frazier Z, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009e.pdf">Computational representation of
  biological systems</a>. <cite>Methods in Molecular Biology</cite>
  541: 535-549, 2009.
  
  <li> Guerquin M, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009d.pdf">The Bioverse API and Web
  Application</a>. <cite>Methods in Molecular Biology</cite> 541:
  511-534, 2009.
  
  <li> Rashid I, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009c.pdf">Inferring molecular interaction
  pathways from eQTL data</a>. <cite>Methods in Molecular
  Biology</cite> 541: 211-223, 2009.
  
  <li> Wichadakul D, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009b.pdf">Prediction and
  integration of regulatory and protein-protein
  interactions</a>. <cite>Methods in Molecular Biology</cite> 541:
  101-143, 2009.
	   
  <li> McDermott J, Wang J, Yu J, Wong GSK,
  <b>Samudrala R</b>. In Rao GP, Wagner C, Singh RK,
  editors. <a href="publications/samudrala_2008g.pdf">Prediction and
  annotation of plant protein interaction
  networks</a>.  <cite>Application of Genomics and Bioinformatics in
  Plants</cite> (Studium Press) 207-238, 2008.
    
  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2007d.pdf">Bioinformatic characterization
  of plant networks</a>. <cite>Proceedings of the Asia Pacific Conference
  on Plant Tissue Culture and Agrobiotechnology</cite>, 2007.      
    
  <li> Chang AN, McDermott J, Guerquin M, Frazier Z, <b>Samudrala
  R</b>. <a href="publications/samudrala_2006b.pdf">Integrator: Interactive
  graphical search of large protein
  interactomes over the Web</a>. <cite>BMC Bioinformatics</cite> 7:
  146, 2006.
   
  <li> McDermott J, Bumgarner RE, <b>Samudrala R</b>. <a
  href="publications/samudrala_2005i.pdf">Functional annotation from
  predicted protein interaction networks</a>.
  <cite>Bioinformatics</cite> 21: 3217-3226, 2005. *

  <li> McDermott J, Guerquin M, Frazier Z, Chang AN, <b>Samudrala R</b>.
  <a href="publications/samudrala_2005h.pdf">BIOVERSE: Enhancements to the
  framework for structural, functional, and contextual annotations of
  proteins and proteomes</a>. <cite>Nucleic Acids Research</cite>
  33: W324-W325, 2005. *
  
  <li> Chang AN, McDermott J, <b>Samudrala R</b>.
  <a href="publications/samudrala_2005d.pdf">An enhanced java
  graph applet interface for visualizing
  interactomes</a>. <cite>Bioinformatics</cite> 21: 1741-1742, 2005.
   
  <li> Yu J, Wang J, Lin W, Li S, Li H, Zhou J, ..., McDermott J,
  <b>Samudrala R</b>, Wang J, Wong GK.  <a
  href="publications/samudrala_2005a.pdf">The genomes of <i>Oryza
  sativa</i>: A history of duplications</a>.  <cite>PLoS
  Biology</cite> 3: e38, 2005. *
  
  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2004a.pdf">Enhanced functional information from
  protein networks</a>. <cite>Trends in
  Biotechnology</cite> 22: 60-62, 2004. *

  <li> McDermott J, <b>Samudrala R</b>.  <a
  href="publications/samudrala_2003d.pdf">BIOVERSE: Functional, structural,
  and contextual annotation of proteins and
  proteomes</a>. <cite>Nucleic Acids Research</cite> 31:
  3736-3737, 2003. *

  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2002e.pdf">The Bioverse: An object-oriented
  genomic database and webserver written in
  Python</a>. In <cite>Proceedings of the conference on Objects in
  Bio- & Chem-Informatics</cite>, 2002.

  <li> <a href="http://bioverse.org">Bioverse framework</a>    

  <li> <a href="http://protinfo.org">Protinfo structure,  function, and interaction prediction server</a>
</ol>


</div>
<div class="section" id="interaction">
<h2>Interaction<a class="headerlink" href="#interaction" title="Permalink to this headline">¶</a></h2>
<p>Methods for predicting interactions between molecules.</p>

<ol>
  <li> Kittichotirat W, Guerquin M, Bumgarner RE, <b>Samudrala R</b>.
  Protinfo PPC: <a href="publications/samudrala_2009i.pdf">A web
  server for atomic level prediction of protein complexes</a>.
  <cite>Nucleic Acids Research</cite> 37: W519-W525, 2009. *
  
  <li> Bernard B, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009g.pdf">A
  generalized knowledge-based discriminatory function for biomolecular
  interactions</a>.  <cite>Proteins: Structure, Function, and
  Bioinformatics</cite> 76: 115-128, 2009. *

  <li> McDermott J, Bumgarner RE, <b>Samudrala R</b>. <a
  href="publications/samudrala_2005i.pdf">Functional annotation from
  predicted protein interaction networks</a>.
  <cite>Bioinformatics</cite> 21: 3217-3226, 2005. *

  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2004a.pdf">Enhanced functional information from
  protein networks</a>. <cite>Trends in
  Biotechnology</cite> 22: 60-62, 2004.    

  <li> <a href="http://bioverse.org">Bioverse framework</a>    
</ol>

</div>
<div class="section" id="function">
<h2>Function<a class="headerlink" href="#function" title="Permalink to this headline">¶</a></h2>
<p>Generally applicable methods for predicting protein function from sequence and/or structure.</p>
<ol>
   
  <li> McDermott JE, Corrigan A, Peterson E, Oehmen C,
  Niemann G, Cambronne ED, Sharp D, Adkins JN, <b>Samudrala R</b>,
  Heffron F.  Computational prediction of type III and IV secreted
  effectors in Gram-negative bacteria.  <i>Infection and Immunity</i>
  79: 23-32, 2010.
    
  <li> Horst JA, Wang K, Horst OV, Cunningham ML, <b>Samudrala
  R</b>. <a href="publications/samudrala_2010f.pdf">Disease risk of
  missense mutations using structural inference from predicted
  function</a>. <cite>Current Protein & Peptide Science</cite> 11:
  573-588, 2010.
  
  <li> Horst J, <b>Samudrala R</b>. A protein sequence meta-functional
  signature for calcium binding residue prediction. <cite>Pattern
  Recognition Letters</cite> 31: 2103-2112, 2010. *
  
  <li> <b>Samudrala R</b>, Heffron F, McDermott J. <a
  href="publications/samudrala_2009h.pdf"><i>In silico</i> identification of
  secreted effectors in <i>Salmonella typhimurium</a></i>. <cite>PLoS
  Pathogens</cite> 5: e1000375, 2009. *

  <li> Wang K, Horst J, Cheng G, Nickle D, <b>Samudrala R</b>. <a
  href="publications/samudrala_2008f.pdf">Protein meta-functional signatures
  from combining sequence, structure, evolution and amino acid
  property information</a>. <cite>PLoS Computational Biology</cite> 4:
  e1000181, 2008. *
  
  <li> Wang K, <b>Samudrala R</b>. <a
  href="publications/samudrala_2006h.pdf">Incorporating background frequency
  improves entropy-based residue conservation measures</a>. <cite>BMC
  Bioinformatics</cite> 7: 385, 2006.
   
  <li> Wang K, <b>Samudrala R</b>. <a
  href="publications/samudrala_2006e.pdf">Automated functional
  classification of experimental and predicted protein
  structures</a>. <cite>BMC Bioinformatics</cite> 7: 278, 2006. *
    
  <li> Cheng G, Qian B, <b>Samudrala R</b>, Baker D. <a
  href="publications/samudrala_2005m.pdf">Improvement in
  protein functional site prediction by distinguishing structural and
  functional constraints on protein family evolution using
  computational design</a>. <cite>Nucleic Acids Research</cite>
  33: 5861-5867, 2005. *

  <li> McDermott J, Bumgarner RE, <b>Samudrala R</b>. <a
  href="publications/samudrala_2005i.pdf">Functional annotation from
  predicted protein interaction networks</a>.
  <cite>Bioinformatics</cite> 21: 3217-3226, 2005. *
    
  <li> Wang K, <b>Samudrala R</b>.  <a
  href="publications/samudrala_2005f.pdf">FSSA: A novel method for
  identifying functional signatures from structural
  alignments</a>. <cite>Bioinformatics</cite> 21: 2969-2977, 2005. *

  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2004a.pdf">Enhanced functional information from
  protein networks</a>. <cite>Trends in Biotechnology</cite> 22: 60-62, 2004.
  
  <li> <a href="http://protinfo.org">Protinfo structure, function, and interaction prediction server</a>

</ol>


</div>
<div class="section" id="structure">
<h2>Structure<a class="headerlink" href="#structure" title="Permalink to this headline">¶</a></h2>
<p><strong>De novo protein structure prediction</strong></p>
<p>The basic paradigm is to sample the conformational space exhaustively or
semi-exhaustively such that native-like conformations are observed. These conformations
are selected using the all-atom based scoring functions. Some methods have had
good success in the CASP blind prediction experiments.</p>
<ol>

  <li> Laurenzi A, Hung L-H, <b>Samudrala R</b>. Structure prediction
  of partial length protein sequences: applications in foldability
  prediction and EST annotation. <cite>International Journal of
  Molecular Sciences</cite> 214: 14892-14907, 2013.
  
  <li> Liu T, Horst J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009j.pdf">A novel method for
  predicting and using distance constraints of high accuracy for
  refining protein structure prediction</a>.  <cite>Proteins: Structure,
  Function, and Bioinformatics</cite> 77: 220-234, 2009. *

  <li> Horst J, <b>Samudrala R</b>.  <a
  href="publications/samudrala_2009l.pdf">Diversity of protein structures
  and difficulties in fold recognition: The curious case of Protein
  G</a>. <cite>F1000 Biology Reports</cite> 1:69, 2009. *
   
  <li> Hung L-H, Ngan S-C, <b>Samudrala R</b>. <a
  href="publications/samudrala_2007a.pdf"><i>De novo</i>
  protein structure prediction</a>. In Xu Y, Xu D, Liang J, editors.
  <cite>Computational Methods for Protein Structure Prediction and
  Modeling</cite> 2: 43-64, 2007.
   
  <li> Hung L-H, Ngan S-C, Liu T, <b>Samudrala R</b>.
  <a href="publications/samudrala_2005g.pdf">PROTINFO: New algorithms for
  enhanced protein structure prediction</a>. <cite>Nucleic Acids
  Research</cite> 33: W77-W80, 2005. *
  
  <li> Hung L-H, <b>Samudrala R</b>. <a
  href="publications/samudrala_2003c.pdf">PROTINFO: Secondary and
  tertiary protein structure prediction</a>. <cite>Nucleic Acids
  Research</cite> 31: 3296-3299, 2003. *

  <li> <b>Samudrala R</b>, Levitt M. <a
  href="publications/samudrala_2002d.pdf">A comprehensive analysis of 40
  blind protein structure predictions</a>. <cite>BMC Structural
  Biology</cite> 2: 3-18, 2002. *
  
  <li> <b>Samudrala R</b>. <a
  href="publications/samudrala_2002a.pdf">Lessons from blind protein structure
  prediction experiments</a>. In Grohima M, Selvaraj S,
  eds. <cite>Recent Research Developments in Protein Folding,
  Stability, and Design</cite>, 123-139, 2002.
  
  <li> Xia Y, Huang ES, Levitt M, <b>Samudrala R</b>. <a
  href="publications/samudrala_2000f.pdf"><i>Ab initio</i>
  construction of protein tertiary structures using a hierarchical
  approach</a>. <cite>Journal of Molecular Biology</cite>, 300:
  171-185, 2000. *
   
  <li> <b>Samudrala R</b>, Xia Y, Levitt M. Huang ES. <a
  href="publications/samudrala_1999b.pdf"><i>Ab initio</i> prediction of
  protein structure using a combined hierarchical
  approach</a>. <cite>Proteins: Structure, Function, and Genetics</cite>
  S3: 194-198, 1999. *

  <li> Huang ES, <b>Samudrala R</b>, Ponder JW. <a
  href="publications/dg_casp3_paper.pdf">Ab initio protein
  structure prediction results using a simple distance geometry
  method</a>. <i>unpublished</i>.
  
  <li> Huang ES, <b>Samudrala R</b>, Ponder JW. <a
  href="publications/samudrala_1999c.pdf"><i>Ab initio</i> fold
  prediction of small helical proteins using distance geometry and
  knowledge-based scoring functions</a>. <cite>Journal of Molecular
  Biology</cite> 290:267-281, 1999.
  
  <li> Huang ES, <b>Samudrala R</b>, Ponder JW. <a
  href="publications/samudrala_1998d.pdf">Distance geometry generates native-like
  folds for small helical proteins using the consensus distances of
  predicted protein structures</a>. <cite>Protein Science</cite> 7:
  1998-2003, 1998.

  <li> <b>Samudrala R</b>, Xia Y, Levitt M, Huang ES. <a
  href="publications/samudrala_1999a.pdf">A combined approach for <i>ab
  initio</i> construction of low resolution protein tertiary
  structures from sequence</a>. In Altman R, Dunker K, Hunter L,
  Klein T, Lauderdale K, eds. <cite>Proceedings of the Pacific
  Symposium on Biocomputing</cite> 505-516, 1999.

  <li> <a href="http://protinfo.org">Protinfo structure, function, and interaction prediction server</a>
</ol>

<h3> Comparative modelling of protein structure </h3>

<p> Handling the problem of context sensitivity in protein
structures. Some methods have had good success in the CASP blind
prediction experiments.  </p>

<ol>

  <li> Bondoc JMG, Gutka HJ, Almutairi MM, Patwell R, Rutter MW, Wolf
  NM, <b>Samudrala R</b>, Mehboob S, Dementiev A, Abad-Zapatero C,
  Movahedzadeh F. Rv0100, a proposed acyl carrier protein in
  Mycobacterium tuberculosis: expression, purification and
  crystallization. Corrigendum. <cite>Acta Crystallograpica F
  Structural Biology Communications</cite> 76: 192-193, 2020.
    
  <li> Bondoc JMG, Gutka HJ, Almutairi MM, Patwell R, Rutter MW, Wolf
  NM, <b>Samudrala R</b>, Mehboob S, Movahedzadeh F. Rv0100, a
  proposed acyl carrier protein in Mycobacterium tuberculosis:
  expression, purification and crystallization. <cite>Acta
  Crystallograpica F Structural Biology Communications</cite>. 75:
  646-651, 2019.
  
  <li> Kittichotirat W, Guerquin M, Bumgarner RE, <b>Samudrala R</b>.
  Protinfo PPC: <a href="publications/samudrala_2009i.pdf">A web server for
  atomic level prediction of protein complexes</a>.  <cite>Nucleic
  Acids Research</cite> 37: W519-W525, 2009.

  <li> Liu T, Horst J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009j.pdf">A novel method for
  predicting and using distance constraints of high accuracy for
  refining protein structure prediction</a>.  <cite>Proteins: Structure,
  Function, and Bioinformatics</cite> 77: 220-234, 2009.
  
  <li> Liu T, Guerquin M, <b>Samudrala R</b>. <a
  href="publications/samudrala_2008c.pdf">Improving the accuracy of
  template-based predictions by mixing and matching between initial
  models</a>. <cite>BMC Structural Biology</cite> 8: 24, 2008.
   
  <li> Hung L-H, Ngan S-C, Liu T, <b>Samudrala R</b>.
  <a href="publications/samudrala_2005g.pdf">PROTINFO: New algorithms for
  enhanced protein structure prediction</a>. <cite>Nucleic Acids
  Research</cite> 33: W77-W80, 2005.

  <li> Hung L-H, <b>Samudrala R</b>. <a
  href="publications/samudrala_2003c.pdf">PROTINFO: Secondary and
  tertiary protein structure prediction</a>. <cite>Nucleic Acids
  Research</cite> 31: 3296-3299, 2003. *

  <li> <b>Samudrala R</b>, Levitt M. <a
  href="publications/samudrala_2002d.pdf">A comprehensive analysis of 40
  blind protein structure predictions</a>. <cite>BMC Structural
  Biology</cite> 2: 3-18, 2002. *
  
  <li> <b>Samudrala R</b>. <a
  href="publications/samudrala_2002a.pdf">Lessons from blind protein structure
  prediction experiments</a>. In Grohima M, Selvaraj S,
  eds. <cite>Recent Research Developments in Protein Folding,
  Stability, and Design</cite>, 123-139, 2002.

  <li> <b>Samudrala R</b>, Moult J.  <a
  href="publications/samudrala_1998b.pdf">A graph-theoretic algorithm for
  comparative modelling of protein structure</a>. <cite>Journal of
  Molecular Biology</cite> 279: 287-302, 1998. *
  
  <li> <b>Samudrala R</b>, Moult J.  <a
  href="publications/samudrala_1997b.pdf">Handling context-sensitivity
  in protein structures using graph theory: <i>bona fide</i>
  prediction</a> <cite>Proteins: Structure, Function, and
  Genetics</cite> 29S: 43-49, 1997. *
  
  <li> <b>Samudrala R</b>. <a href="publications/samudrala_1997a.pdf">A
  graph-theoretic solution to the context-sensitivity problem in
  protein structure prediction</a>. <i>Ph.D. thesis</i>, 1997.
  
  <li> <b>Samudrala R</b>, Pedersen JT, Zhou H, Luo R, Fidelis K,
  Moult J.  <a href="publications/samudrala_1995a.pdf">Confronting the
  problem of interconnected structural changes in the comparative
  modelling of proteins</a>.  <cite>Proteins: Structure, Function, and
  Genetics</cite> 23: 327-336, 1995.

  <li> <a href="http://protinfo.org">Protinfo structure, function, and interaction prediction server</a>
</ol>

<h3> Protein structure from combining theory and experiment </h3>

<p> Use the structure prediction methods described below with
experimental data to produce better results. </p>

<ol>
  <li> Hung L-H, <b>Samudrala R</b>. <a
  href="publications/samudrala_2006i.pdf">An automated assignment-free
  Bayesian approach for accurately identifying proton contacts from
  NOESY data</a>. <cite>Journal of Biomolecular NMR</cite> 36:
  189-198, 2006. *
  
  <li> Hung L-H, <b>Samudrala R</b>. <a
  href="publications/samudrala_2003c.pdf">PROTINFO: Secondary and
  tertiary protein structure prediction</a>. <cite>Nucleic Acids
  Research</cite> 31: 3296-3299, 2003. *

  <li> Hung L-H, <b>Samudrala R</b>. <a
  href="publications/samudrala_2003a.pdf">Accurate and automated
  classification of protein secondary structure with
  PsiCSI</a>. <cite>Protein Science</cite> 12: 288-295, 2003. *

  <li> <a href="http://protinfo.org">Protinfo structure, function, and interaction prediction server</a>
</ol>

<h3> Scoring/discriminatory functions for protein structure prediction </h3>

<p> We primarily use an all-atom distance dependent conditional
probability discriminatory function that is surprisingly accurate at
selecting correct from incorrect protein conformations. It is used
both for <i>ab initio</i> prediction and comparative modelling. We
also use a number of other scoring functions as filters, and also
develop databases of incorrect conformations ("decoys") to help
evaluate scoring functions. </p>

<ol>

  <li> Moughon S, <b>Samudrala R</b>. LoCo: a new backbone-only
  scoring function for protein structure prediction. <cite>BMC
  Bioinformatics</cite> 12: 368, 2011.
    
  <li> Bernard B, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009g.pdf">A
  generalized knowledge-based discriminatory function for biomolecular
  interactions</a>.  <cite>Proteins: Structure, Function, and
  Bioinformatics</cite> 76: 115-128, 2009.

  <li> Ngan S-C, Hung L-H, Liu T, <b>Samudrala R</b>. <a
  href="publications/samudrala_2007h.pdf">Scoring functions for <i>de
  novo</i> protein structure prediction revisited</a>.  <cite>Methods
  in Molecular Biology</cite> 413: 243-282, 2007.
    
  <li> Liu T, <b>Samudrala R</b>. <a
  href="publications/samudrala_2006g.pdf">The effect of experimental
  resolution on the performance of knowledge-based discriminatory
  functions for protein structure selection</a>. <cite>Protein
  Engineering, Design and Selection</cite> 19: 431-437, 2006.
    
  <li> Ngan S-C, Inouye M, <b>Samudrala R</b>. <a
  href="publications/samudrala_2006a.pdf">A knowledge-based
  scoring function based on residue triplets for protein structure
  prediction</a>. <cite>Protein Engineering, Design and
  Selection</cite> 19: 187-193, 2006.
   
  <li> Wang K, Fain B, Levitt M, <b>Samudrala R</b>.  <a
  href="publications/samudrala_2004d.pdf">Improved protein structure
  selection using decoy-dependent discriminatory
  functions</a>. <cite>BMC Structural Biology</cite> 4: 8, 2004. *
    
  <li> <b>Samudrala R</b>, Levitt M. <a
  href="publications/samudrala_2000d.pdf">Decoys 'R' Us: A database of
  incorrect protein
  conformations for evaluating scoring functions</a>. <cite>Protein
  Science</cite>, 9: 1399-1401, 2000.

  <li> Huang ES, <b>Samudrala R</b>, Park BH. <a
  href="publications/samudrala_2000c.pdf">Scoring functions
  for <i>ab initio</i> folding</a>. In Walker J, Webster D,
  eds. <cite>Predicting Protein Structure: Methods and Protocols</cite>
  Humana Press, 2000.

  <li> <b>Samudrala R</b>, Moult J.  <a
  href="publications/samudrala_1998a.pdf">An all-atom distance-dependent
  conditional probability discriminatory function for protein
  structure prediction</a>.  <cite>Journal of Molecular Biology</cite>
  275: 893-914, 1998. *

  <li> <a href="http://dd.compbio.org">Decoys 'R' Us database</a>
</ol>

<h3> Side chain prediction </h3>

<p> There are two papers in this area. The first is a work on exactly
what it is that primarily determines side chain conformational
preferences in proteins.  The main thrust here is the use of the
discriminatory function to select the most probable side chain
rotamers given a large number of possible conformations.  The second
paper compares different methods for side chain prediction. </p>
  
<ol>
  <li> <b>Samudrala R</b>, Huang ES, Koehl P, Levitt M. <a
  href="publications/samudrala_2000b.pdf">Side chain construction on
  non-native main chains using an all-atom discriminatory
  function</a>. <cite>Protein Engineering</cite>, 7: 453-457, 2000.
  
  <li> <b>Samudrala R</b>, Moult J.  <a
  href="publications/samudrala_1998c.pdf">Determinants of side chain
  conformational preferences in protein
  structures</a>.  <cite>Protein Engineering</cite> 11: 991-997, 1998.
  
</ol>


</div>
<div class="section" id="infrastructure">
<h2>Infrastructure<a class="headerlink" href="#infrastructure" title="Permalink to this headline">¶</a></h2>
<p>We prefer to make our clusters from cheap components that can be readily
discarded, and prefer to completely decentralize our systems. Also
included in this category are algorithms developed to handle the scientific problems
we face.</p>
<ol>

  <li> Hung L-H, <b>Samudrala R</b>. fast_protein_cluster: parallel and
  optimized clustering of large scale protein modeling
  data. <cite>Bioinformatics</cite> 30: 1774-1776, 2014.

  <li> Hung L-H, <b>Samudrala R</b>. Accelerated protein structure
  comparison using TM-score-GPU. <cite>Bioinformatics</cite> 28:
  2191-2192, 2012.

  <li> Hung LH, Guerquin M, <b>Samudrala R</b>. GPU-Q-J, a fast method
  for calculating root mean square deviation (RMSD) after optimal
  superposition. <cite>BMC Research Notes</cite> 4: 97, 2011.
    
  <li> Frazier Z, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009e.pdf">Computational representation of
  biological systems</a>. <cite>Methods in Molecular Biology</cite>
  541: 535-549, 2009.
  
  <li> Guerquin M, McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2009d.pdf">The Bioverse API and Web
  Application</a>. <cite>Methods in Molecular Biology</cite> 541:
  511-534, 2009.

  <li> <b>Samudrala R</b>. <a href="publications/firewall.pdf">Taking the
  cost out of firewalls</a>. <cite>LinuxWorld Magazine</cite> 1: 58-59, 2003. 
  
  <li> <b>Samudrala R</b>. <a
  href="http://www.ibiblio.org/pub/Linux/docs/HOWTO/other-formats/pdf/Cluster-HOWTO.pdf">Linux
  Cluster HOWTO</a>, 2003.

  <li> McDermott J, <b>Samudrala R</b>. <a
  href="publications/samudrala_2002e.pdf">The Bioverse: An object-oriented
  genomic database and webserver written in
  Python</a>. In <cite>Proceedings of the conference on Objects in
  Bio- & Chem-Informatics</cite>, 2002.

  <li> <b>Samudrala R</b>. <a href="publications/raid.pdf">Installing and
  using RAID</a>. In Danesh A, Gautam D, eds. <cite>Special Edition
  Using Linux System Administration</cite>, Que Publishing, 2000.
</ol>

</div>
</div>


           </div>
           
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright Not asserted for this work

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>