

<!DOCTYPE html>
<html class="writer-html5" lang="ls -ltr" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>SARS-CoV-2 and COVID-19 related work &mdash; Tuttle and Associates - Beyond Warming 1.1.1 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>
        <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home" alt="Documentation Home"> Tuttle and Associates - Beyond Warming
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <!-- Local TOC -->
              <div class="local-toc"><ul>
<li><a class="reference internal" href="#">SARS-CoV-2 and COVID-19 related work</a><ul>
<li><a class="reference internal" href="#files">Files</a></li>
<li><a class="reference internal" href="#methods">Methods</a></li>
<li><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
</ul>
</div>
            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Tuttle and Associates - Beyond Warming</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>SARS-CoV-2 and COVID-19 related work</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/SARS-CoV-2_and_COVID-19_related_work.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="sars-cov-2-and-covid-19-related-work">
<h1>SARS-CoV-2 and COVID-19 related work<a class="headerlink" href="#sars-cov-2-and-covid-19-related-work" title="Permalink to this headline">¶</a></h1>
<h2>CANDO platform putative drug candidates against COVID-19</h2><p>These pages contains information on drug candidates generated by the CANDO multitarget
shotgun drug discovery, repurposing, and design platform against SARS-CoV-2 and COVID-19.
<strong>Last update of predictions: May 8, 2020; CANDO version v2.</strong></p>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>Interpreting this data without understanding the context and how the
platform works could be hazardous to your health ( literally and figuratively )!</p>
</div>
<p>These are a set of initial and preliminary predictions of human approved drugs and
compounds likely to inhibit the SARS coronavirus 2 (COVID-19/SARS-CoV-2).
..
The predictions were made using the <a class="reference external" href="http://protinfo.org/cando">CANDO</a> platform.
CANDO works well in expert hands, but these predictions are a first pass application
optimizing across the SARS proteome first, then using the top scoring methods to
create predictions against the COVID-19 proteome.
..
Refinement of the predictions based on considering the protein targets carefully
is still in progress, and attempts are being made to validate the predictions at
the bench.</p>
<p>The accuracy of the CANDO platform is between 12%-35% depending on the number of
compounds evaluated both in terms of retrospective benchmarking and
prospective in vitro validation ( see <a class="reference internal" href="#references"><span class="std std-ref">References</span></a> and links below ).
Indeed, from twelve prospective in vitro validation studies across ten
indications, 58/163 (35%) top ranking predictions validated using the CANDO
platform (v1) or its components had comparable or better activity relative to a
standard treatment (if available, or micromolar or better inhibitors of the pathogenic
system, in this case COVID-19). The validated predictions represent potential novel
repurposed therapies for indications such as dengue, dental caries, diabetes, herpes,
lupus, malaria, and tuberculosis. The logic is that since these are compounds generally
approved for human use, the predictions that are verified in vitro are good candidates
for off-label use human studies to evaluate efficacy and ultimately,
represent potential new therapies for a particular indication.</p>
<div class="section" id="files">
<h2>Files<a class="headerlink" href="#files" title="Permalink to this headline">¶</a></h2>
<table class="colwidths-given docutils align-default" id="id2">
<caption><span class="caption-text">Results</span><a class="headerlink" href="#id2" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 17%" />
<col style="width: 83%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>March 5, 2020</p></td>
<td><p>&lt;<a class="reference external" href="http://compbio.buffalo.edu/protinfo/cando/results/covid19/mar052020/">http://compbio.buffalo.edu/protinfo/cando/results/covid19/mar052020/</a>&gt;</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>March 16, 2020</p></td>
<td><p>&lt;<a class="reference external" href="http://compbio.buffalo.edu/protinfo/cando/results/covid19/mar162020/">http://compbio.buffalo.edu/protinfo/cando/results/covid19/mar162020/</a>&gt;</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>April 10, 2020</p></td>
<td><p>&lt;<a class="reference external" href="http://compbio.buffalo.edu/protinfo/cando/results/covid19/apr102020/">http://compbio.buffalo.edu/protinfo/cando/results/covid19/apr102020/</a>&gt;</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>May 18, 2020</p></td>
<td><p>&lt;<a class="reference external" href="http://compbio.buffalo.edu/protinfo/cando/results/covid19/may182020/">http://compbio.buffalo.edu/protinfo/cando/results/covid19/may182020/</a>&gt;</p></td>
</tr>
</tbody>
</table>
<p>The manifests for each directory corresponding to a particular round of predictions
are available above. Currently, four lists of predictions were generated using three
approaches:
- A homology based approach using known actives against the original SARS-CoV virus.
- A de novo method involving the top predicted binding affinities of approved drugs against structures from the COVID-19 proteome.
- A protein structure agnostic approach using only fingerprint similarities between the SARS actives. The SARS actives were extracted from two high-throughput screens from Shen et al. (2019), Dyall et al. (2014), and an additional broad-spectrum active from Cao et al. (2015).</p>
<p>For the homology based approach ( canpredict-homology-human-all.txt ), our canpredict
module was utilized in which we tallied the number of times certain drugs/compounds
were ranked highly in the ranked similarity lists to drugs known to have activity against
SARS-CoV (38 compounds total, which themselves had a back recovery rate aka accuracy
of roughly 50%). The set of proteins used for this approach was 5,317 human protein
structures from the PDB.
This list includes experimental/investigational compounds selected from a total of
8,696, including 1,979 approved
drugs ( <a class="reference external" href="http://go.drugbank.com/">Drugbank</a> ),
and only includes
compounds with a consensus score of at least 2 (second column -&gt; &quot;score1&quot;).
This same method was used for the protein structure independent method in which
structural fingerprints were used instead of protein- compound interaction scores
(ecfp4-2048-bit_vect_tani-cp_human.txt)</p>
<p>For the <em>de novo</em> method
( ecfp10-covid19_coach-int_vect-dice-pscoreXpercentile-CTD1860set-top225.txt ),
the predictions are based on generating interactions between known drugs and viral
proteins in a holistic manner: the feature being predicted is really the binding
or &quot;stickiness&quot; of small molecules to viral proteins. This in turn is expected to
turn up some inhibitors (of both the proteins and the pathogen) which in turn would
lead to clinical efficacy. Some of the predictions are obviously incorrect or
inappropriate, but the unedited output of the software is what is provided; it may
be possible to refine the predictions further manually. These predictions only include
drugs with at least one associated indication from the
<a class="reference external" href="http://ctdbase.org/">Comparative Toxicogenomics Database</a>, which served as a proxy for &quot;approved&quot; status
(some unapproved drugs are still present, which are indicated by a 'false' in
the 'approved' column). The March 16, 2020 list of predictions is largely the same as
the one from March 5, 2020, but only includes approved drugs according to
DrugBank's mapping (the scoring method is identical).</p>
<p>Some of the interesting predictions from the March 5, 2020 round include
chloroquine and other antimalarials at rank 35-40 (approach 1), ACE inhibitors at
rank 19-21 (approach 2), remdesivir at rank 46 (approach 2), and amprenavir and
other HIV protease inhibitors at rank 48-50 (approach 2), all of which have either
shown or believed to have efficacy against SARS-CoV-2 and are undergoing (multiple)
clinical trials to demonstrate efficacy. Therefore, some of the other higher ranked
drugs in our lists are also worth evaluating, with the potential payoff of choice,
greater efficacy, and reduced cost if shown to inhibit SARS-CoV-2 in vitro.</p>
<p>Both prediction methods use the same output schema for listing candidates.</p>
<dl class="field-list simple">
<dt class="field-odd">Column 1 ( 'rank' )</dt>
<dd class="field-odd"><p>Is Rank of the prediction.</p>
</dd>
<dt class="field-even">Column 2 ( 'score1' )</dt>
<dd class="field-even"><p>Is the primary score used to rank. In the case of the homology method, this would be the number of times the drug showed up in the topN most similar drugs to the SARS actives. For the de novo method, this is the number of protein interactions above the set threshold.</p>
</dd>
<dt class="field-odd">Column 3 ( 'score2' )</dt>
<dd class="field-odd"><p>Is the tiebreaker score - for the homology method, this is average rank of each of the 'score1' votes (lower is better), while for the de novo method, this is the sum of each interaction above the threshold (higher is better).</p>
</dd>
<dt class="field-even">Column 4 ( 'id' )</dt>
<dd class="field-even"><p>Is the in-house ID number assigned to CANDO drugs within the package.</p>
</dd>
<dt class="field-odd">Column 5 ( 'approved' )</dt>
<dd class="field-odd"><p>Is whether or not the drug is FDA approved according to DrugBank; a '*' or '+' indicates that the compound is associated with the query indication according to the input drug-disease mapping (in our case, from the Comparative Toxicogenomics Database). The de novo method will include associated drugs for the indication in the output list even if they are unapproved as a reference.</p>
</dd>
<dt class="field-even">Column 6 ( 'name' )</dt>
<dd class="field-even"><p>is simply the generic name of the drug/compound.</p>
</dd>
</dl>
</div>
<div class="section" id="methods">
<h2>Methods<a class="headerlink" href="#methods" title="Permalink to this headline">¶</a></h2>
<p>The publications that describe the CANDO platform in further detail are given below.
The platform is based on a large number of methods developed by us and by others for
protein and proteome structure, function, and interaction prediction. In this specific
case: For the homology based approach rankings, the CANDO platform was used as
described in the publications except for the changes specified above. For the de novo
method, eighteen proteins encoded by the SARS-CoV-2 genome were modelled to obtain
tertiary structures using I-TASSER. Modelling was accomplished by inferring homology
to structures in the PDB (typically SARS proteins) determined by x-ray diffraction.
The CANDO pipeline was used to to obtain predicted interaction scores between these
eighteen structures and a library of 8,696 small molecule compounds, which includes
1,860 human ingestible drugs. The compounds with the strongest predicted interaction
scores ( &gt; 0.9 ) and those that bound the most frequently to these 18 protein
structures using various scoring protocols are ranked and reported.</p>
</div>
<div class="section" id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this headline">¶</a></h2>

<ul>
<!--  <li> Schuler J, benchmarking paper. -->

  <li> Mangione W, Falls Z, Chopra G, <b>Samudrala R</b>. <a
  href="https://www.biorxiv.org/content/10.1101/845545v2">cando.py:
  Open source software for predictive bioanalytics of large scale
  drug-protein-disease data</a>.  <cite>bioRxiv</cite> 845545; doi:
  https://doi.org/10.1101/845545, 2020.
  
</ul>


<p> CANDO platform development, recent significant publications, in
order of citation importance for this work: </p>

<ul>

  <li> Minie M, Chopra G, Sethi G, Horst JA, White G, Roy A, Hatti K,
  <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2014b.pdf">CANDO and
  the infinite drug discovery frontier</a>.  <cite>Drug Discovery
  Today</cite> 19: 1353-1363, 2014.

  <li> Mangione W, <b>Samudrala R.</b> Identifying protein features
  responsible for improved drug repurposing accuracies using the CANDO
  platform: Implications for drug design. <i>Molecules</i> 24: 167,
  2019.

  <li> Falls Z, Mangione W, Schuler J, <b>Samudrala R.</b> Exploration
  of interaction scoring criteria in the CANDO platform. <i>BMC
  Research Notes</i> 12: 318, 2019.

  <li> Schuler J, <b>Samudrala R</b>. Fingerprinting CANDO: Increased
  accuracy with structure and ligand based shotgun drug
  repurposing. <cite>ACS Omega</cite> 4: 17393-17403, 2019.

  <li> Fine J, Lackner R, <b>Samudrala R</b>, Chopra G. Computational
  chemoproteomics to understand the role of selected psychoactives in
  treating mental health indications. <cite>Scientific Reports</cite>
  9, 1315, 2019.

  <li> Chopra C, Kaushik S, Elkin PL, <b>Samudrala R</b>. Combating
  Ebola with repurposed therapeutics using the CANDO
  platform. <cite>Molecules</cite> 21: 1537, 2016. [Roughly similar to
  the present study described above (which is done with the latest
  version of CANDO and its corresponding enhancements).]

  <li> Chopra G, <b>Samudrala R</b>. Exploring polypharmacology in
  drug discovery and repurposing using the CANDO
  platform. <cite>Current Pharmaceutical Design</cite> 22: 3109-3123
  2016.

  <li> Sethi G, Chopra G, <b>Samudrala R</b>. <a
  href="http://compbio.org/publications/samudrala_2015a.pdf">Multiscale
  modelling of relationships between protein classes and drug behavior
  across all diseases using the CANDO platform</a>. <cite>Mini Reviews
  in Medicinal Chemistry</cite>, 15: 705-717, 2015.
    
</ul>


<h4>Further</h4><p>See also all our
<a class="reference external" href="http://compbio.buffalo.edu/publications/therapeutics.html">publications related to therapeutic discovery</a> as</p>
<blockquote>
<div><p>well as a comprehensive list of all our publications.</p>
<p><a class="reference external" href="http://protinfo.org/cando">CANDO</a></p>
</div></blockquote>
</div>
</div>


           </div>
           
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright Not asserted for this work

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>